manifest.json

```diff
@@ -246,6 +266,7 @@
             {
               "dockerfile": "src/ubuntu/14.04",
               "os": "linux",
+              "osVersion": "trusty",
```

Comment:
The difference in versioning scheme here is unfortunate.  The Dockerfile paths and FROM statements are using the version number syntax while the osVersion is using the version code name.  Is it possible to unify these?  Can the osVersion be ubuntu14.04?

Reply:
Sure.  Also updated xenial to ubuntu16.04.